src/Control/Eff.hs

```diff
@@ -1,6 +1,33 @@
-module Control.Eff ( module Internal
-                   , module OpenUnion
-                   ) where
+{-# LANGUAGE ExplicitNamespaces #-}
 
-import Control.Eff.Internal as Internal hiding (Lift(..), lift, runLift)
+-- | A monadic library for implementing effectful computation in a modular way.
+--
+-- This module provides the @Eff@ monad - the base type for all effectful
+-- computation.
+-- The @Member@ typeclass is the main interface for describing the .
+--
+-- Consult the @Control.Eff.QuickStart@ module and the readme for gentle
+-- introductions.
+--
+-- To use extensible effects effectively some language extensions are
+-- necessary/suggested.
+--
+-- @
+-- {-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
@power-fungus do these need to be escaped somehow? i don't see them in the generated docs via `make doc`. specifically, i observe
```
To use extensible effects effectively some language extensions are necessary/suggested.

{--}
{--}
{--}
```